src/frontend/Sider/Behandling/BehandlingTabsInnhold.tsx

```diff
@@ -83,57 +84,59 @@ const BehandlingTabsInnhold = () => {
             behandling={behandling}
             behandlingErRedigerbar={behandlingErRedigerbar}
         >
-            <Tabs value={aktivFane} onChange={(e) => håndterFaneBytte(e as FanePath)}>
-                <StickyTablistContainer>
-                    <TabsList>
-                        {behandlingFaner.map((tab) =>
-                            tab.erLåst ? (
-                                <DisabledTab
-                                    key={tab.path}
-                                    value={tab.path}
-                                    label={tab.navn}
-                                    icon={tab.ikon}
-                                />
-                            ) : (
-                                <Tabs.Tab
-                                    key={tab.path}
-                                    value={tab.path}
-                                    label={tab.navn}
-                                    icon={tab.ikon}
-                                />
-                            )
-                        )}
-                        {kanSetteBehandlingPåVent && !statusPåVentRedigering && (
-                            <Tabsknapp>
-                                <Button
-                                    size={'small'}
-                                    onClick={() => settStatusPåVentRedigering(true)}
-                                    variant="secondary"
-                                >
-                                    Sett på vent
-                                </Button>
-                            </Tabsknapp>
-                        )}
-                        {behandlingErRedigerbar && <HamburgermenyBehandling />}
-                    </TabsList>
-                </StickyTablistContainer>
-
-                {!toggleKanSaksbehandle && (
-                    <Alert variant={'error'}>Mulighet for å saksbehandle er skrudd av</Alert>
-                )}
-                <SettPåVentContainer
-                    statusPåVentRedigering={statusPåVentRedigering}
-                    settStatusPåVentRedigering={settStatusPåVentRedigering}
-                />
-
-                {behandlingFaner
-                    .filter((fane) => !fane.erLåst)
-                    .map((tab) => (
-                        <BehandlingContent key={tab.path} value={tab.path}>
-                            {tab.komponent(behandling.id)}
-                        </BehandlingContent>
-                    ))}
-            </Tabs>
+            <ManglendeBrevVariablerProvider>
```

Comment:
Bare ManglendeBrevVariablerProvider som er lagt til i denne filen

Reply:
Den burde kunne legges inn i de komponenter som brukes `BrevMeny`, 
`Brev` og `FrittståendeBrev`

Reply:
fixed: https://github.com/navikt/tilleggsstonader-sak-frontend/pull/661/commits/0fa853d96f16373a1f2639e2c8404cd8247040a6

Reply:
Her tenkte jeg at det kunde være fint å ha den langt ned i treet. Nå vil man dele context i behandling og frittstående brev, uten at den nullstilles mellom de

Reply:
ah, da misforsto jeg. Du mener at "ManglendeBrevVariablerProvider" bør legges in 2 steder: Så lavt nede i tre-strukturen som mulig?

Reply:
fixed: 145b10785914eb68f53faa64fa500063b4ca1e14